src/scene/materials/standard-material.js

```diff
@@ -181,6 +181,8 @@ let _params = new Set();
  * alternative to specular color to save space. With metaless == 0, the pixel is assumed to be
  * dielectric, and diffuse color is used as normal. With metaless == 1, the pixel is fully
  * metallic, and diffuse color is used as specular color instead.
+ * @property {boolean} useSpecularF0 Use specular map to apply color tint to specular reflections
```

Comment:
Since this flag enables specular for metalness, perhaps we can call this `metalnessUseSpecular`? Not sure devs will understand meaning of F0.